docs/src/AerosolActivation.md

```diff
@@ -285,3 +285,100 @@ The CloudMicrophysics package offers an advanced feature for predicting the aero
 In addition, we provide functionality and a demonstration for re-training the free parameters of the traditional ARG activation scheme using Ensemble Kalman Processes. This functionality is showcased in `test/aerosol_activation_calibration.jl`.
 
 Using ML emulators or calibration with Ensemble Kalman Processes for predicting aerosol activation is provided through an extension to the main package. This extension will be loaded with `CloudMicrophysics.jl` if both `MLJ.jl`, `DataFrames.jl`, and `EnsembleKalmanProcesses.jl` are loaded by the user as well.
+
+## Modification for Local Supersaturation with Preexisting Hydrometeors
+
+The original ARG activation scheme assumes adiabatic ascent with no preexisting cloud or ice particles, which is a valid approximation near cloud base. As a result, the ARG formulation is traditionally applied only at cloud base, where no vapor has yet condensed and no hydrometeors exist to affect the supersaturation evolution.
+
+However, in many applications—such as large-eddy simulations or cloud-resolving models—it is desirable to apply the activation scheme locally, throughout the vertical extent of the cloud. In such cases, preexisting cloud droplets and ice crystals act as persistent vapor sinks, suppressing supersaturation and inhibiting further activation.
+
+To relax the cloud-base-only constraint and extend ARG into a local operator, we propose modifying the supersaturation tendency equation by incorporating the effects of these preexisting hydrometeors.
+
+### Modified Supersaturation Budget
+
+We augment the supersaturation tendency with additional vapor sinks:
+
+```math
+\frac{dS}{dt} = \alpha w - \gamma \frac{d\chi}{dt}_{activation} - \gamma \frac{d\chi}{dt}_{liquid} - \gamma \frac{d\chi}{dt}_{ice},
+```
+
+where the new sink terms are:
+
+- Liquid sink:
+```math
+\frac{d\chi}{dt}_{liquid} = 4 \pi \rho_w N_l r_l G_l S,
+```
+
+- Ice sink:
+```math
+\frac{d\chi}{dt}_{ice} = 4 \pi \rho_i N_i r_i G_i S_i,
+```
+
+with
+```math
+S_i = \xi (S + 1) - 1, \quad \xi = \frac{e_{s,i}}{e_{s,\ell}}.
+```
+
+Here:
+
+- ``N_l``, ``r_l``, and ``G_l`` are the number concentration, radius, and condensational growth coefficient of liquid droplets,
+- ``N_i``, ``r_i``, and ``G_i`` are the corresponding values for ice crystals,
+- ``\rho_w`` and ``\rho_i`` are the densities of liquid water and ice.
+
+We define:
+```math
+K_l = 4 \pi \rho_w N_l r_l G_l, \quad K_i = 4 \pi \rho_i N_i r_i G_i,
+```
+
+yielding the modified supersaturation equation at peak supersaturation:
+```math
+0 = \alpha w - \gamma f(S_{max}) - K_l S_{max} - K_i \big[ \xi (S_{max} + 1) - 1 \big].
```

Comment:
Should Kl and Ki be multiplied by gamma?

Reply:
They include gamma